pkg/modifiers/modifiers.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2024 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package modifiers provides the functionality to parse meta modifiers from text.
+package modifiers
+
+import (
+	"context"
+	"regexp"
+)
+
+var metaValuesKVRegex = regexp.MustCompile(`(?m)^(GUARDIAN_[A-Z0-9_]+)=(.*)$`)
+
+type MetaValues map[string][]string
+
+func ParseBodyMetaValues(ctx context.Context, contents string) MetaValues {
+	metaValues := make(MetaValues, 0)
+
+	matches := metaValuesKVRegex.FindAllStringSubmatch(contents, -1)
+	for _, v := range matches {
+		if _, ok := metaValues[v[1]]; !ok {
```

Comment:
It might help this be more readable if we assign `v[1]` to a variable name, like `indexOfFoo := v[1]`. Just to make this more clear for anyone who hasn't memorized the meaning of the return values from FindAllStringSubmatch 😀

Reply:
Ah, fair, will update.